pylint/checkers/stdlib.py

```diff
@@ -466,8 +466,10 @@ def __init__(self, linter: PyLinter) -> None:
         # synced with the config argument deprecated-modules
 
     def _check_bad_thread_instantiation(self, node: nodes.Call) -> None:
-        if not node.kwargs and not node.keywords and len(node.args) <= 1:
-            self.add_message("bad-thread-instantiation", node=node)
+        if not node.kwargs and "target" not in {key.arg for key in node.keywords}:
+            self.add_message(
+                "bad-thread-instantiation", node=node, confidence=interfaces.HIGH
+            )
```

Comment:
seems HIGH to be the right confidence here, no?

Reply:
I don't see any inference so I'd say yes.